modules/templates/templates/langs/en.json

```diff
@@ -43,5 +43,15 @@
     "fieldUpdateSuccess": "Field updated successfully",
     "fieldUpdateError": "Error updating field",
     "finish-editing": "Please finish editing the the current template before creating a new one"
+  },
+  "button": {
+    "btn-delete-title": "Delete",
+    "btn-edit-title": "Edit",
+    "btn-view-title": "View",
+    "btn-cancel": "Cancel",
+    "btn-save": "Save",
+    "btn-backward": "Back",
+    "btn-skip": "Skip",
+    "btn-play": "Play"
```

Comment:
Eddy Lets simplify here

Just "delete" instead of "btn-delete-title" 

Same for the rest :)